src/backend/joanie/tests/core/test_api_admin_users.py

```diff
@@ -211,3 +211,54 @@ def test_admin_api_user_delete(self):
             "The requested resource was not found on this server.",
             status_code=404,
         )
+
+    def test_admin_api_user_me_no_access(self):
+        """
+        User should see their infos on the me route
+        """
+        admin = factories.UserFactory(is_staff=True, is_superuser=True)
+        self.client.login(username=admin.username, password="password")
+
+        response = self.client.get("/api/v1.0/admin/users/me/")
+
+        self.assertEqual(response.status_code, 200)
+        content = response.json()
+        self.assertEqual(content["id"], str(admin.id))
+        self.assertEqual(content["is_superuser"], True)
+        self.assertEqual(content["is_staff"], True)
+        self.assertEqual(content["has_course_access"], False)
+        self.assertEqual(content["has_organization_access"], False)
```

Comment:
why not completely equal the whole object?
```suggestion
        self.assertEqual(
            response.json(),
            {
                "id": str(admin.id),
                "is_superuser": admin.is_superuser,
                ...
            }
        )
```